lib/Serialization/SILFormat.h

```diff
@@ -268,7 +268,7 @@ namespace sil_block {
                      BCFixed<1>,  // global_init
                      BCFixed<2>,  // inlineStrategy
                      BCFixed<2>,  // side effect info.
-                     BCFixed<2>,  // number of specialize attributes
+                     BCFixed<16>,  // number of specialize attributes
```

Comment:
Can you just make this a `BCVBR<8>` instead? Size doesn't matter so much here, but that way we're not encoding any particular hard limit.

Reply:
@jrose-apple Very good point! I totally forgot about the existence of `BCVBR`